manifests/rdoinfo.pp

```diff
@@ -17,6 +17,11 @@
     mode   => '0755',
     owner  => 'rdoinfo',
   } ->
+  exec { 'ensure home contents belong to rdoinfo':
+    command => 'chown -R rdoinfo:rdoinfo /home/rdoinfo',
```

Comment:
@amoralej  @javierpena
Why is this not the same as for promoter, with file recurse => true ?